python: modem-firmware-1.3+: device_credentials_installer (#18)
nRFCloud/utils: python/modem-firmware-1.3+/README.md

  --port PORT           Specify which serial port to open, otherwise pick from list (default: None)
  --id_str ID_STR       Device ID to use instead of UUID. Will be a prefix if used with --id_imei (default: )
  --id_imei             Use IMEI for device ID instead of UUID. Add a prefix with --id_str (default: False)

>   --coap                Install both the AWS root CA cert as well as the CoAP server root CA cert

Reviewer: ```suggestion
  --coap                Install the CoAP server root CA cert in addition to the AWS root CA cert
```
